### ISSUE TITLE
[Bug]: Broken/missing Hostile Neural Network (HNN) predictions

### ISSUE THREAD

**Issue by zentaos** [AllTheMods/ATM-8#1808]
### Possible Fixes

Yes

### Modpack Version

1.1.0

### What happened?

Squid:
![image](https://github.com/AllTheMods/ATM-8/assets/988308/59fcac1d-fabf-4e37-bbd9-2f7efa67ea05)
![image](https://github.com/AllTheMods/ATM-8/assets/988308/1714e702-4df7-48d6-9127-1b063a2a1e30)

Quests:
![image](https://github.com/AllTheMods/ATM-8/assets/988308/64fdb788-d99f-4ab0-b9cd-e99bda5937c9)
![image](https://github.com/AllTheMods/ATM-8/assets/988308/5f81c7be-fe2f-40cc-8448-05a59bdfbab8)
![image](https://github.com/AllTheMods/ATM-8/assets/988308/85b70836-07e2-4d96-90d8-3d228cc86c1f)
![image](https://github.com/AllTheMods/ATM-8/assets/988308/b0157ca8-1522-44f7-b4e6-144b89229474)

### latest.log

-

**Comment by Mitchell5200** [comment]
yeah, that's on me, Fixed in dev, in Commit https://github.com/AllTheMods/ATM-8/pull/1802/commits/402e6c44aeef39708189f4b28cbcfc787bcc969c

**Comment by zentaos** [comment]
Nice thanks ❤

**Comment by zentaos** [comment]
Tested, works 👍

**Comment by Nimurian** [comment]
I am experiencing the same issue - are there any instructions on how to implement this? Thanks in advance!
